static/styles/night_mode.scss

```diff
@@ -337,15 +337,23 @@ on a dark background, and don't change the dark labels dark either. */
 
     #streamlist-toggle,
     #userlist-toggle {
-        border-color: hsla(199, 33%, 46%, 0.2);
+        color: inherit;
+        border-color: hsla(0, 0%, 0%, 0.6);
     }
 
-    #streamlist-toggle-button,
-    #userlist-toggle-button {
+    #streamlist-toggle-button {
         color: inherit;
         background-color: inherit;
     }
 
+    #userlist-toggle-button {
+        color: hsl(221, 9%, 54%);
```

Comment:
I had no idea how to match inheritance up with the gear icon on this (or even where the color for the night mode gear icon was defined).  This `hsl` value was found using the mozilla color dropper on the gear icon itself.